components/accept-financial-contributions/StripeOrBankAccountPicker.js

```diff
@@ -137,9 +203,41 @@ class StripeOrBankAccountPicker extends React.Component {
             </Container>
           </Flex>
         </Box>
+        {(isBankAccountAlreadyThere || stripeAccount) && (
+          <StyledButton
+            fontSize="13px"
+            minHeight="36px"
+            mt={4}
+            minWidth={'145px'}
+            onClick={() => {
+              const host = LoggedInUser ? LoggedInUser.collective : hostOrganization;
+              addHost(collective, host);
+              Router.pushRoute('accept-financial-contributions', {
+                slug: router.query.slug,
+                path: router.query.path,
+                state: 'success',
+              }).then(() => window.scrollTo(0, 0));
+            }}
+          >
+            <FormattedMessage id="Pagination.Next" defaultMessage="Next" />
+          </StyledButton>
+        )}
       </Flex>
     );
   }
 }
 
-export default withRouter(StripeOrBankAccountPicker);
+const getBankAccountQuery = graphql(gql`
+  query Collective($collectiveSlug: String) {
+    Collective(slug: $collectiveSlug) {
+      id
+      slug
+      name
+      settings
+    }
+  }
+`);
+
+const addGraphQL = compose(getBankAccountQuery);
```

Comment:
1) Double check that and use a more common syntax (`const bankAccountQuery =`, `const addBankAccountQuery =`. `compose` is not needed.
2) The information in the query are really generic, have you considered re-using an existing query instead?

Reply:
It is! I think I did actually copy an existing query but then took out all of the fields I wasn't using in an attempt to be more.... 'frugal' with the code? But you're right it probably makes sense to just reuse the other query even if it has fields I don't need.

Reply:
Maybe the one coming from `getCollectivePageQuery` should be enough?